io-engine/src/subsys/nvmf/admin_cmd.rs

```diff
@@ -158,15 +158,15 @@ pub fn create_snapshot(
             | (nvme_cmd_cdw11_get_val(cmd) as u64) << 32
     };
     let snapshot_name = Lvol::format_snapshot_name(&lvol.name(), snapshot_time);
-    let snap_param = SnapshotParams::new(
+    let mut snap_param = SnapshotParams::new(
         Some(lvol.name()),
         Some(lvol.name()),
         Some(lvol.name()),
```

Comment:
Since we now support 'tx_id', can't we generate a GUID here to make transactions unique ?

Reply:
> Since we now support 'tx_id', can't we generate a GUID here to make transactions unique ?

Accepted, modified. Missed this part. Thanks for finding.